plugins/api-docs/src/components/OpenApiDefinitionWidget/OpenApiDefinition.tsx

```diff
@@ -147,9 +179,27 @@ export const OpenApiDefinition = ({ definition }: OpenApiDefinitionProps) => {
     return () => clearTimeout(timer);
   }, [definition, setDef]);
 
+  // Due to a bug in the swagger-ui-react component, the component needs
+  // to be created without content first.
+  const [plgins, setPlugins] = useState<any[] | undefined>([{}]);
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      if (!!plugins)
+        setPlugins(
+          plugins
+            ?.filter(plugin =>
+              Object.keys(supportedSwaggerPlugins).includes(plugin),
+            )
+            ?.map(plugin => supportedSwaggerPlugins[plugin]),
+        );
+    }, 0);
+    return () => clearTimeout(timer);
+  }, [plugins, setPlugins]);
```

Comment:
This is quite unnecessary as far as I can see, you'd really just want to compute the set of plugins straight away rather than wrapping it up with `useState`/`useEffect`